### PR TITLE
[FOR DISCUSSION]: DevTools on Flutter for Web as a separate package

### PR DESCRIPTION
**Pull request by DaveShuckerow**
This is an alternate approach to building out devtools on Flutter by creating a separate package for the Flutter UI.

### CODE REVIEW

File: packages/devtools_flutter/lib/src/connect_screen.dart [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    return Screen(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text('Connect', style: textTheme.headline),
+          const Divider(),
+          const Padding(padding: EdgeInsets.only(top: 10.0)),
+          Text('Connect to a running app', style: textTheme.body2),
+          Text('Enter a port or URL to a running Dart or Flutter application',
+              style: textTheme.caption),
```
Comment (jacob314) [inline packages/devtools_flutter/lib/src/connect_screen.dart]: use flutter style of trailing commas here and elsewhere

File: packages/devtools_flutter/lib/src/screen.dart [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+class Screen extends StatelessWidget {
```
Comment (jacob314) [inline packages/devtools_flutter/lib/src/screen.dart]: Is each screen going to be a tab within the main application or are you envisioning a different setup?

File: packages/devtools_flutter/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+author: Dart Team <misc@dartlang.org>
+homepage: https://github.com/flutter/devtools
+
+environment:
+  sdk: '>=2.3.0 <3.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
```
Comment (jacob314) [inline packages/devtools_flutter/pubspec.yaml]: is that plan to depend of flutter_app with a dev dependency?

File: packages/devtools_flutter/lib/src/screen.dart (hunk 2 above)
Comment (DaveShuckerow) [reply packages/devtools_flutter/lib/src/screen.dart]: See the doc comments in https://github.com/flutter/devtools/pull/1106/files#diff-bcf4904b4fdc024eca0bbbe11b73ee4aR3

File: packages/devtools_flutter/pubspec.yaml (hunk 3 above)
Comment (DaveShuckerow) [reply packages/devtools_flutter/pubspec.yaml]: With a path dependency.

**Comment by DaveShuckerow** [comment]
I'm going to close this one in favor of #1106, which is branched from this PR.

**Pull request closed without merging**